src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -431,7 +432,11 @@ export class SavedObjectsService
       ).toPromise();
 
       this.logger.info('Starting saved objects migrations');
+      const migrationSpan = apm.startSpan('saved_objects.migration');
```

Comment:
Can you specify type for this span? Helps with annotating the breakdown metrics for the transaction with specified span duration. https://www.elastic.co/guide/en/apm/agent/nodejs/current/metrics.html#metrics-span.self_time.sum  

```js
apm.startSpan('saved_objects.migration', 'migration');
```